app.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "Nango",
+    "description": "Get OAuth tokens for APIs - fast & secure",
+    "website": "https://www.namgo.dev",
+    "repository": "https://github.com/NangoHQ/nango",
+    "logo": "https://raw.githubusercontent.com/NangoHQ/nango/master/docs/static/img/nango-favicon.svg",
+    "stack": "container",
+    "image": "nango.docker.scarf.sh/nangohq/nango-server",
+    "env": {
+        "NANGO_SECRET_KEY": {
```

Comment:
We should remove the `NANGO_SECRET_KEY` env because it's only used to secure instances (cf. [docs](instance)), which makes the setup a bit more complicated. I think it should remain off by default.

Sorry, I didn't notice it was added for the Render Blueprint. We should remove it from there too.